src/pybind/mgr/dashboard/tox.ini

```diff
@@ -5,8 +5,7 @@ envlist =
     fix,
     check,
     run,
-    apidocs,
-    openapi-{check,fix, doc}
+    openapi-{check, fix, doc}
```

Comment:
Don't we need this for the code [here](https://github.com/ceph/ceph/blob/5bb9f33c9158b61a0561db2be07272a83e43af6f/src/pybind/mgr/dashboard/tox.ini#L179)?

Reply:
@aaSharma14  `openapi-doc` is not removed, only `apidocs` (which in fact is running `py3`).